SCons/Script/Main.xml

```diff
@@ -943,6 +944,51 @@ Example:
 SetOption('max_drift', 0)
 </example_commands>
 </summary>
+
+
 </scons_function>
 
+    <scons_function name="ValidateOptions">
+        <arguments signature="global">
+            ([throw_exception=False])
+        </arguments>
+
+        <summary>
+            <para>
+                Check that all the options specified on the command line are either defined by SCons itself
+                or defined by &f-link-AddOption;.
```

Comment:
feels like something is missing - maybe add the word "calls" at the end of the sentence?

Reply:
ok